frontend/src/components/data-visualization/StatisticsSection/utils.ts

```diff
@@ -0,0 +1,10 @@
+export const NUMBER_FORMAT = (value: number) => value.toFixed(0);
+export const PERCENTAGE_FORMAT = (value: number) => `${NUMBER_FORMAT(value)}%`;
+export const NTH_FORMAT = (value: number) => {
+  const lastDigit = value % 10;
+  let suffix = "th";
+  if (lastDigit === 1) suffix = "st";
+  if (lastDigit === 2) suffix = "nd";
+  if (lastDigit === 3) suffix = "rd";
+  return `${NUMBER_FORMAT(value)}${suffix}`;
+};
```

Comment:
I put it here because I thought it wouldn't really be used in any context other than with this component. Were you thinking it would be better to just have a `StatisticsUtils.ts` in the general folder instead?

Reply:
i think we just put all the utils there, regardless of whether they're used in any other context, just as an organizational thing! but i can live with it either way :D